@kiva/kv-components/vue/KvCartModal.vue

```diff
@@ -0,0 +1,339 @@
+									class="
+                    tw-grid tw-content-center tw-justify-center
+                    tw-ml-auto
+                    tw-w-6 tw-h-6 tw--m-2
+                    hover:tw-text-action-highlight
+                  "
+									@click.stop="hide('close-x')"
+								>
+									<kv-material-icon
+										class="tw-w-3 tw-h-3"
+										:icon="mdiClose"
+									/>
+									<span class="tw-sr-only">Close</span>
+								</button>
+							</div>
+						</div>
+
+						<!-- body -->
+						<div
+							id="kvCartModalBody"
+							ref="kvCartModalBody"
+							class="
+								tw-flex
+                tw-gap-2
+								tw-px-2.5 md:tw-px-4
+							"
+							style="height: 3.75rem;"
+						>
+							<div>
+								<kv-borrower-image
+									class="tw-rounded loan-image"
+									:alt="borrowerImage.alt"
+									:aspect-ratio="borrowerImage.aspectRatio"
+									:default-image="borrowerImage.defaultImage"
+									:hash="borrowerImage.hash"
+									:images="borrowerImage.images"
+									:photo-path="photoPath"
+								/>
+							</div>
+							<div class="tw-flex tw-items-center tw-justify-between tw-w-full">
+								<div class="tw-flex tw-flex-col tw-h-full tw-justify-between">
+									<p>{{ borrowerName }}</p>
+									<p class="tw-p-1 tw-text-center tw-rounded tw-bg-secondary tw-text-h5">
+										{{ borrowerCountry }}
+									</p>
+								</div>
+								<p> ${{ amount }}</p>
+							</div>
+						</div>
+
+						<!-- controls -->
+						<div
+							ref="controlsRef"
+							class="
+								tw-flex-shrink-0 tw-flex tw-justify-end tw-gap-x-2.5
+								tw-p-2.5 md:tw-px-4 md:tw-pb-4
+							"
+						>
+							<kv-button
+								class="tw-w-full"
+								href="/checkout"
+							>
+								View basket ({{ basketCount }})
+							</kv-button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</transition>
+</template>
+
+<script>
+import {
+	ref,
+	toRefs,
+	computed,
+	nextTick,
+	watch,
+	onBeforeUnmount,
+	onMounted,
+} from 'vue-demi';
+import { mdiClose, mdiCheckCircle } from '@mdi/js';
+import { useFocusTrap } from '@vueuse/integrations/useFocusTrap';
+import { hideOthers as makePageInert } from 'aria-hidden';
+import { lockScroll, unlockScroll } from '../utils/scrollLock';
+import { lockPrintSingleEl, unlockPrintSingleEl } from '../utils/printing';
+import KvButton from './KvButton.vue';
+import KvMaterialIcon from './KvMaterialIcon.vue';
+import KvBorrowerImage from './KvBorrowerImage.vue';
+
+/**
+ * Based on KvLightbox functionality
+ */
+
+export default {
+	components: {
+		KvMaterialIcon,
+		KvButton,
+		KvBorrowerImage,
+	},
+	props: {
+		/**
+		 * Whether the dialog is open or not
+		 * */
+		visible: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * The dialog has no close X button, clicking the screen does not close,
+		 * pressing ESC does not close.
+		 * */
+		preventClose: {
+			type: Boolean,
+			default: false,
+		},
+		basketCount: {
```

Comment:
Might as well add jsdoc comments to all props for consistency, and the format should have one fewer asterisk on the props above this one.

```
/**
 * Based on KvLightbox functionality
 */
```